Deploy-Infrastructure.ps1

```diff
@@ -69,10 +71,11 @@ $provisionReqs = @(
   @{
     Name     = "Terraform Apply"
     Describe = "Apply Terraform plan"
-    Test     = { Test-Path "./out/azurek8s" }
+    Test     = { Test-Path "./out/azurek8s" } # TODO: probe infra for test
```

Comment:
This test won't let you redeploy infrastructure if you've already done so.  I'm going to file an issue for a better test here.